### PR TITLE
fix shorts regex + fix broken test

### PR DESCRIPTION
**Pull request by ChunkyProgrammer**
This allows length to be properly parsed from shorts.

Without this change: shorts that were uploaded minutes or seconds would cause an error as well as would cause errors when using from certain regions (ex: some ips for other countries would return data that was not in english so the regex would fail)

### CODE REVIEW

**Comment by ChunkyProgrammer** [comment]
not sure why it's failing exactly

**Comment by GilgusMaximus** [comment]
I guess here we can just apply the same code logic then as over at the other module

**Comment by GilgusMaximus** [comment]
@ChunkyProgrammer use npm to do the package install and then the ci should run smoothly

**Comment by ChunkyProgrammer** [comment]
> @ChunkyProgrammer use npm to do the package install and then the ci should run smoothly

done

**Review by GilgusMaximus — DISMISSED** [review]
Same as over at yt-trending-scraper

File: app/helper.js [hunk 1]
```diff
@@ -238,12 +237,17 @@ class YoutubeGrabberHelper {
...
-          const regexMatch = video.title.accessibility.accessibilityData.label.match(shortsRegex)
-          lengthSeconds = parseInt(regexMatch[2])
-          durationText = '0:' + (lengthSeconds.toString().padStart(2, '0'))
-          if (regexMatch[3] === 'minute') {
+          const accessibilityData = video.title.accessibility.accessibilityData.label
+          const numbersAndSpacesRegex = /[^0-9\s]/g
+          const numbersOnly = accessibilityData.replace(numbersAndSpacesRegex, '').trim().split(' ').filter(number => {
+            return number !== ''
+          })
+          lengthSeconds = parseInt(numbersOnly[numbersOnly.length - 2])
```
Comment (PikachuEXE) [inline app/helper.js:245]: I can't understand this logic until I see an example value :S
```suggestion
          // Example value for 1 minute: `xxxxxx`
          // Example value for less than 1 minute: `xxxxxx`
          lengthSeconds = parseInt(numbersOnly[numbersOnly.length - 2])
```

**Review by PikachuEXE — CHANGES_REQUESTED** [review]
Also I have no idea how to test this

**Review by PikachuEXE — APPROVED** [review]

**Comment by ChunkyProgrammer** [comment]
@absidue updated description

**Review by absidue — APPROVED** [review]

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — APPROVED** [review]

**Pull request merged**